tests/sentry/workflow_engine/processors/test_detector.py

```diff
@@ -876,6 +877,7 @@ def test_activity_not_supported(self) -> None:
             get_detector_by_event(event_data)
 
     def test_no_detector_id(self) -> None:
```

Comment:
let's update this test name as well, so it's clear that the test is now looking for an issue stream detector rather than an error.